python-api/gstswitch/controller.py

```diff
@@ -339,10 +329,8 @@ def click_video(self, xpos, ypos, width, height):
         try:
             conn = self.connection.click_video(xpos, ypos, width, height)
             res = conn.unpack()[0]
-            if res is True:
-                print "Click video: xpos:%s ypos:%s width:%s height:%s" % (
-                    str(xpos), str(ypos), str(width), str(height))
-            else:
+            if not res is True:
```

Comment:
Here you should use;
```python
if res is not True:
```

Coming from other languages you might expect the above to be equivalent to the following, but it instead does what the english says;
```python
if res is (not True):
```